### ISSUE TITLE
Failed to restore the playback from S3 on BYT with ALC5651.

### ISSUE THREAD

**Issue by markyang** [thesofproject/sof#648]
**Summary:**
  Failed to restore the playback from S3 on BYT with ALC5651 everytime.
  sof-logger was blocked by [sof/issues/642](https://github.com/thesofproject/sof/issues/642)
  
**dmesg:**
```
[  228.771792] CPU2 is up
[  228.771822] smpboot: Booting Node 0 Processor 3 APIC 0x6
[  228.772633]  cache: parent cpu3 should not be sleeping
[  228.773113] CPU3 is up
[  228.773932] ACPI: Waking up from system sleep state S3
[  228.787756] byt_gpio INT33FC:00: restored pin 62 conf0 0x2003cc81
[  228.787767] byt_gpio INT33FC:00: restored pin 63 conf0 0x2083cc81
[  228.787776] byt_gpio INT33FC:00: restored pin 64 conf0 0x2003cc81
[  228.787785] byt_gpio INT33FC:00: restored pin 65 conf0 0x2083cc81
[  228.787911] byt_gpio INT33FC:00: restored pin 99 conf0 0x2003cc01
[  228.788023] byt_gpio INT33FC:02: restored pin 0 conf0 0x260bcc80
[  228.939632] acpi LNXPOWER:04: Turning OFF
[  228.939703] acpi LNXPOWER:03: Turning OFF
[  228.939744] acpi LNXPOWER:02: Turning OFF
[  228.939771] acpi LNXPOWER:01: Turning OFF
[  228.939842] acpi LNXPOWER:00: Turning OFF
[  228.939893] OOM killer enabled.
[  228.939894] Restarting tasks ... 
[  228.951277] sof-audio sof-audio: PCM: stream 0 dir 0 DMA position 15168 DAI position 782112
[  228.951361] sof-audio sof-audio: PCM: stream 0 dir 0 DMA position 15168 DAI position 782112
[  228.951432] sof-audio sof-audio: PCM: stream 0 dir 0 DMA position 15168 DAI position 782112
[  228.955856] done.
[  228.955954] video LNXVIDEO:00: Restoring backlight state
[  228.955964] PM: suspend exit
[  229.558422] sof-audio sof-audio: PCM: stream 0 dir 0 DMA position 15168 DAI position 782112
[  230.067629] sof-audio sof-audio: pcm: trigger stream 0 dir 0 cmd 0
[  230.370616] sof-audio sof-audio: error: ipc timed out for 0x60050000 size 0xc
[  230.370868] sof-audio sof-audio: error: unexpected fault 0xffffffff trace 0xffffffff
[  230.370884] sof-audio sof-audio: error: waking up any trace sleepers
[  230.370941]  Passthrough: ASoC: trigger FE failed -110
[  230.370994] sof-audio sof-audio: pcm: free stream 0 dir 0
[  230.674604] sof-audio sof-audio: error: ipc timed out for 0x60030000 size 0xc
[  230.674856] sof-audio sof-audio: error: unexpected fault 0xffffffff trace 0xffffffff
[  230.674872] sof-audio sof-audio: error: waking up any trace sleepers
[  230.675395] sof-audio sof-audio: pcm: close stream 0 dir 0
[  231.748186] igb 0000:03:00.0 enp3s0: igb: enp3s0 NIC Link is Up 100 Mbps Full Duplex, Flow Control: RX/TX
```

**Test steps:**
1. aplay -Dhw:0,0 -c2 -fS16_LE -vv -i ~/Music/18_WAV_48000Hz_stereo_16bit.wav
2. echo mem > /sys/power/state

**Test env:**
sof master: 920a3bce
sof tool:   ad3f3e4f
kernel sof-dev: 405e866b
tplg: test-ssp2-mclk-0-I2S-volume-s16le-s16le-48k-19200k-codec.tplg-ad3f3e4f

**Log:**
[dmesg.log](https://github.com/thesofproject/sof/files/2627826/dmesg.log)

**Comment by mengdonglin** [comment]
@markyang  I think SOF doesn't support power management on BYT. So this bug can be invalid.

@lgirdwood @ranj063 @xiulipan Please correct me if I'm wrong.

**Comment by lgirdwood** [comment]
@markyang S3 not supported on BYT Minnowboard.

**Comment by markyang** [comment]
@xiulipan I've created a new issue to follow the 'sof-audio hw params failed' error( thesofproject/sof/issues/675 ). we can close this one.

**Comment by lgirdwood** [comment]
@mengdonglin @markyang please only re-open Minnowboard S3 issues if you have a BIOS resource that can fix the issue otherwise they are distraction.
